Rakefile.rb

```diff
@@ -67,3 +67,24 @@ def json_write(file, json)
     puts "Pull request #{pull_request.number} is outdated. (Newest pull request is #{pull_request.superseded_by.number})"
   end
 end
+
+desc "Post a summary of the pull request that's currently being built"
+task :pull_request_summary do
+  if ENV['TRAVIS_PULL_REQUEST'] == 'false'
```

Comment:
Yep, good point. I'll merge this now so I can shut off the Heroku bits, then split that up as a separate pull request.